_data-prepper/monitoring.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Monitoring
+nav_order: 33
+---
+
+# Monitoring Data Prepper with metrics
+
+Data Prepper can be monitored with metrics using [Micrometer](https://micrometer.io/). There are two types of metrics: JVM and system metrics and plugin metrics. [Prometheus](https://prometheus.io/) is used as the default metrics backend.
```

Comment:
Can we turn this around to be active voice? For example, "You can monitor your Data Prepper metrics using Micrometer. The two types of metrics that you can monitor are _JVM and system metrics_ and _plugin metrics_" (I don't know about the italics, but otherwise it's hard to understand that JVM and system go together. May be my inexperience with the technology, though.)

Reply:
How about:
```suggestion
To see how Data Prepper is performing, you can monitor it with the following metrics:
- JVM and system metrics
- Plugin metrics
[Micrometer](https://micrometer.io/) captures metrics from Data Prepper and publishes them to the [Prometheus](https://prometheus.io/) backend.
```